### PR TITLE
TodoApp example now uses db.

### PR DESCRIPTION
**Pull request by matijaSos**

### CODE REVIEW

File: waspc/cli/Command/Compile.hs [hunk 1]
```diff
@@ -14,12 +19,23 @@ import qualified StrongPath             as SP
 import qualified Util.IO
 
 
+compile :: Command ()
```
Comment (matijaSos) [inline waspc/cli/Command/Compile.hs:22]: It is helpful for me as dev to be able to just compile sometimes without other stuff, so I added this command as we discussed.

File: waspc/cli/Command/Compile.hs [hunk 2]
```diff
@@ -14,12 +19,23 @@ import qualified StrongPath             as SP
...
+compile = do
+    waspRoot <- findWaspProjectRootDirFromCwd
```
Comment (Martinsos) [inline waspc/cli/Command/Compile.hs:24]: waspProjectDir?

File: waspc/examples/todoApp/ext/actions.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
-export const createTask = (task, context) => {
+const prisma = new Prisma.PrismaClient()
+
+export const createTask = async (task, context) => {
   if (Math.random() < 0.5) {
     throw new HttpError(400, 'Failed to create task, random error!')
   }
-  state.tasks = [...(state.tasks || []), task]
+
+  const newTask = await prisma.task.create({
```
Comment (Martinsos) [inline waspc/examples/todoApp/ext/actions.js:13]: All right cool :D!

File: waspc/examples/todoApp/ext/queries.js [hunk 4]
```diff
@@ -1,10 +1,18 @@
 import HttpError from '@wasp/core/HttpError.js'
+import Prisma from '@prisma/client'
+
+// TODO(matija): is it ok to create a new instance in every file? See if it needs to be a
+// singleton somewhere, if it isn't already.
+const prisma = new Prisma.PrismaClient()
```
Comment (Martinsos) [inline waspc/examples/todoApp/ext/queries.js:6]: Hmmm good question, although I guess it is probably ok but hard to be sure.

File: waspc/examples/todoApp/todoApp.wasp [hunk 5]
```diff
@@ -4,15 +4,15 @@ app todoApp {
 
 entityPSL Project {=psl
     id      Int     @id @default(autoincrement())
-    tasks   Task[]
+    //tasks   Task[]
```
Comment (Martinsos) [inline waspc/examples/todoApp/todoApp.wasp:7]: Why this got commented out? Maybe leave a comment regarding that?

**Review by Martinsos — APPROVED** [review]

**Pull request merged**